README.md

```diff
@@ -1,174 +1,94 @@
-how many gestures the model has to learn.
-
-### Recognising gestures
-
-The model will attempt to predict which gesture is being performed immediately after training. Go to the model tab to
-configure the feedback actions, which will be taken when the model makes a prediction.
-
-### Configuring prediction and feedback
-
-In the model tab several options become available once the model has been trained. These can be changed to alter the
-behaviour of the platform and micro:bits.
-
-#### Confidence threshold
-
-Determines how confident the model has to be for a feedback action to be taken for the given gesture. This is a slider
-located to the right of the label and can be adjusted to make the model more sensitive, the default value is 80%, and
-can be changes for each gesture.
-
-Immediately besides it, a confidence-meter shows how confident the model currently is and will change as you perform
-gestures.
-
-#### Changing the LEDs for output micro:bit
-
-Determines which LEDs on the display of the micro:bit will be lit up when the gesture is detected. This is
-
-#### Sound feedback
-
-Determines which sound clip will be played when a gesture is recognised. Depending on which version of micro:bit you
-have, the sound will be output on the speaker of the micro:bit or on your computer, if the micro:bit doesn't support
-speaker audio.
-
-#### I/O feedback
-
-Select the I/O pin you wish to turn on in response to a gesture detection. Combine this with circuitry to make gesture
-recognition perform actions outside the platform such as turning on motors or LEDs.
-
-### Dependencies
-
-ML-Machine utilises several packages, most notable of which are the following:
-
-- [Svelte](https://github.com/sveltejs/svelte) provides the web-development framework.
-- [Windi CSS](https://github.com/windicss/windicss) provides our styling tool.
-- [TensorFlow.js](https://github.com/tensorflow/tfjs) provides machine learning capabilities.
-- [Smoothie Charts](https://github.com/joewalnes/smoothie) and [chart.js](https://github.com/chartjs/Chart.js) provides
-  a visualisation kit.
-
-## Contributions
-
-#### Issues
-
-If you experience an issue or have a question regarding the platform please do not hesitate to raise an issue. The
-maintainers will answer to any issues raised as quickly as possible.
+## License
 
-#### Code contributions
+This software is under the MIT open source license.
 
-If you have an improvement on you wish to share with please read the [contributing](CONTRIBUTING.md) page. There you
-will find information about how to contribute. The maintainers will review the changes as soon as possible.
+[SPDX-License-Identifier: MIT](LICENSE)
 
-### Credits
+Significant code is derived from ML-Machine (also MIT licensed) and is (c)
+Center for Computational Thinking and Design at Aarhus University and
+contributors. See individual file copyright notices for more details.
```

Comment:
I know it's somewhat unusual in this kind of section, but could we also make sure we credit Aarhus with the  conceptual work, as I think we're still at a high level using the ML Machine concept.

```suggestion
contributors. See individual file copyright notices for more details.

Conceptually this project draws heavily on the work done by the Center for Computational Thinking and Design at Aarhus University (see [CCTD.dk](https://cctd.au.dk)) and we're hugely grateful for their ongoing support and collaboration.

We're reall
```